core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -981,6 +988,10 @@ class EMConfig {
             " If 0 or negative, the timeout is not applied.")
     var testTimeout = 60
 
+    @Important(2.2)
```

Comment:
this shouldn't be among `Important`. once we decide a good strategy, that ll be default, and not something user would play with

Reply:
as such, then this ption should be moved down in the file, after the block in which Important configs are defined

Reply:
Ok! Thought we were going to allow users to choose if they wanted their tests to be named with the numbered or action strategy, beyond the thesis testing scope

Reply:
Also misunderstood